test/libyul/objectCompiler/verbatim_bug.yul

```diff
@@ -0,0 +1,124 @@
+object "a" {
+    code {
+        let dummy := 0xAABBCCDDEEFF
+        let input := sload(0)
+        let output
+
+        switch input
+        case 0x00 {
+            // Note that due to a bug the following disappeared from the assembly output.
+            output := verbatim_1i_1o(hex"506000", dummy)
+        }
+        case 0x01 {
+            output := 1
+        }
+        case 0x02 {
+            output := verbatim_1i_1o(hex"506002", dummy)
+        }
+        case 0x03 {
+            output := 3
+        }
+
+        sstore(0, output)
+    }
+}
+// ====
+// optimizationPreset: full
+// ----
+// Assembly:
+//     /* "source":87:88   */
+//   0x00
+//     /* "source":81:89   */
+//   dup1
+//   sload
+//     /* "source":139:307   */
+//   dup1
+//   iszero
+//   tag_1
+//   jumpi
+//     /* "source":316:361   */
+//   dup1
+//     /* "source":321:325   */
+//   0x01
+//     /* "source":316:361   */
+//   eq
+//   tag_3
+//   jumpi
+//     /* "source":370:448   */
+//   dup1
+//     /* "source":375:379   */
+//   0x02
+//     /* "source":370:448   */
+//   eq
+//   tag_5
+//   jumpi
+//     /* "source":462:466   */
+//   0x03
+//     /* "source":457:502   */
+//   eq
+//   tag_7
+//   jumpi
+//     /* "source":87:88   */
+//   0x00
+//     /* "source":512:529   */
+//   sstore
+//     /* "source":118:502   */
+//   stop
+//     /* "source":467:502   */
+// tag_7:
+//     /* "source":481:492   */
+//   pop
+//     /* "source":491:492   */
+//   0x03
+//     /* "source":87:88   */
+//   0x00
+//     /* "source":512:529   */
+//   sstore
+//     /* "source":118:502   */
+//   stop
+//     /* "source":380:448   */
+// tag_5:
+//     /* "source":404:438   */
+//   pop
+//   pop
+//     /* "source":45:59   */
+//   0xaabbccddeeff
+//     /* "source":404:438   */
+//   verbatimbytecode_506002
+//     /* "source":87:88   */
+//   0x00
+//     /* "source":512:529   */
+//   sstore
+//     /* "source":118:502   */
+//   stop
+//     /* "source":326:361   */
+// tag_3:
+//     /* "source":340:351   */
+//   pop
+//   pop
+//     /* "source":350:351   */
+//   0x01
+//     /* "source":87:88   */
+//   0x00
+//     /* "source":512:529   */
+//   sstore
+//     /* "source":118:502   */
+//   stop
+//     /* "source":149:307   */
+// tag_1:
+//     /* "source":263:297   */
+//   pop
+//   pop
+//     /* "source":45:59   */
+//   0xaabbccddeeff
+//     /* "source":263:297   */
+//   verbatimbytecode_506000
```

Comment:
Note that in the first commit that adds this test case, there is no occurrance of ``verbatimbytecode_506000`` at all, while it reappears in the second commit with the fix.